src/cider/nrepl/middleware/util/instrument.clj

```diff
@@ -468,6 +468,14 @@
       out
       (into (empty coll) out))))
 
+(defn- instrument-function-call
+  "Instrument a regular function call sexp."
```

Comment:
Here, function call means a sexp that starts with a symbol which is not a macro nor a special form. 
So it includes java methods, but also includes any regular function call. 

I'll clarify that in the docstring. Would you prefer a different function name?